src/main/scala/com/lifeomic/fhirlib/v3/Deserializer.scala

```diff
@@ -1,26 +1,25 @@
 package com.lifeomic.fhirlib.v3
 
+import java.time.format.DateTimeFormatter
+import java.time.{LocalDateTime}
+
 import com.lifeomic.fhirlib.v3.datatypes._
 import com.lifeomic.fhirlib.v3.resources.{Resource, _}
-import org.joda.time.format.ISODateTimeFormat
```

Comment:
If we've removed all joda usages, can we remove it from the POM?

Reply:
Yes